Optionally enable SO_REUSEPORT in SSF listening

asf-stripe opened the pull request:
#### Summary
This PR enables SO_REUSEPORT on the SSF listener if we're configured to run multiple listeners already. This isn't perfect, but it's a reasonable proxy for SO_REUSEPORT'ing.

#### Motivation
we just discovered that veneur can't restart in the configuration we run.


#### Test plan
Rolled this to QA.


#### Rollout/monitoring/revert plan
* merge
* actually release 1.7!

cory-stripe commented:
👍

ChimeraCoder commented:
There's a better fix for this than vendoring the package. Will follow up with a patch.

ChimeraCoder commented:
Here we go. The issue was just that we weren't ensuring that the tag existed locally before we tried to check it out. Now this should work.

asf-stripe commented:
OK, I removed the vendor games and now this PR only does what it said on the tin originally (@ChimeraCoder, I rebased away your 0.5.0 change - think that's all contained in #285?).

ChimeraCoder commented:
lgtm
